contracts/GovernanceToken.sol

```diff
@@ -21,7 +21,7 @@ contract OriginDollarGovernance is
     constructor() initializer {}
 
     function initialize() public initializer {
-        __ERC20_init("Origin Dollar Governance", "OGV");
```

Comment:
I don't think this is going to change the name, since this is in the initializer, and only called early on.

I think we many just want to add a "name()" function like OGV staking has, and return the correct name there.

Reply:
That's my bad. I totally missed that it's in the initializer. Will change it